migrations/sqlite/2019-05-22-124153_add_custom_domains/up.sql

```diff
@@ -0,0 +1,57 @@
+-- add custom_domain to blogs
+CREATE TABLE IF NOT EXISTS "blogs_add_custom_domain" (
```

Comment:
Just wondering why copying the table is required, instead of just adding the column?

Reply:
because [SQLite supports a limited subset of ALTER TABLE](https://www.sqlite.org/lang_altertable.html)

Reply:
Damn, I missed it was SQLite. Thanks.